README.md

```diff
@@ -26,13 +26,26 @@ Non-source downloads such as WAR files and several Linux packages can be found o
 Our latest and greatest source of Jenkins can be found on [GitHub]. Fork us!
 
 # Contributing to Jenkins
+
 Follow the [contributing guidelines](CONTRIBUTING.md) if you want to propose a change in the Jenkins core.
+For more information about participating in the community  contributing to the Jenkins project,
+see [this page](https://www.jenkins.io/participate/).
 
-# Maintainers guidelines
 Documentation for Jenkins core maintainers is in the [maintainers guidelines](docs/MAINTAINERS.adoc).
 
 # News and Website
-All information about Jenkins can be found on our [website]. Follow us on Twitter [@jenkinsci].
+All information about Jenkins can be found on our [website].
```

Comment:
Other sections got an empty line between title and first sentence.

Reply:
```suggestion

All information about Jenkins can be found on our [website].
```